pkg/webui/console/components/webhook-form/index.js

```diff
@@ -87,6 +93,11 @@ const validationSchema = Yup.object().shape({
     .matches(webhookIdRegexp, Yup.passValues(sharedMessages.validateIdFormat))
     .required(sharedMessages.validateRequired),
   format: Yup.string().required(sharedMessages.validateRequired),
+  basic_auth: Yup.object().shape({
```

Comment:
```suggestion
  _basic_auth: Yup.object().shape({
```

We usually mark meta fields (as in fields that do not map directly to final data shape) with an underscore. See my comment below on using encode/decode on the field level to possibly avoid using this altogether.